eventmesh-examples/src/main/java/org/apache/eventmesh/grpc/sub/app/service/SubService.java

```diff
@@ -48,11 +49,15 @@
 @Component
 public class SubService implements InitializingBean {
 
+    private final SubscriptionItem subscriptionItem = new SubscriptionItem();
+    // CountDownLatch size is the same as messageSize in AsyncPublishInstance.java (Publisher)
+    private final CountDownLatch countDownLatch = new CountDownLatch(AsyncPublishInstance.MESSAGE_SIZE);
     private EventMeshGrpcConsumer eventMeshGrpcConsumer;
-
     private Properties properties;
-
-    private final SubscriptionItem subscriptionItem = new SubscriptionItem();
+    private final String localPort = properties.getProperty(SERVER_PORT);
+    private final String url = getURL(localPort, "/sub/test");
+    private final String eventMeshIp = properties.getProperty(ExampleConstants.EVENTMESH_IP);
+    private final String eventMeshGrpcPort = properties.getProperty(ExampleConstants.EVENTMESH_GRPC_PORT);
 
```

Comment:
Please note that there are many other similar changes that need to be reverted, including two types of problems, "redundant indention" and "field/method sequence and formatting", as mentioned in my last review.